.github/workflows/frontend.yml

```
@@ -0,0 +1,35 @@
+name: Frontend Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'frontend/**'
```

I think we should also add this workflow itself.

```suggestion
      - 'frontend/**'
      - '.github/workflows/frontend.yml'
```

---

done, added this workflow to paths.